### PR TITLE
Some tests for SearchResultContainer and ResultsInfo component.

### PR DESCRIPTION
**Pull request by nikomakela**
TH-1075 Added some tests for SearchResultContainer and ResultsInfo component.

### CODE REVIEW

**Comment by nikomakela** [comment]
More related tests in #277

File: src/domain/eventSearch/searchResultList/__tests__/SearchResultsContainer.test.tsx [hunk 1]
```diff
@@ -1,23 +1,60 @@
 import React from 'react';
 
 import { render, screen } from '../../../../test/testUtils';
+import { EventType } from '../../../event/types';
 import SearchResultsContainer from '../SearchResultsContainer';
 
-it('should return no results text if no events is found', async () => {
+it.each([
```
Comment (Sartonon) [inline src/domain/eventSearch/searchResultList/__tests__/SearchResultsContainer.test.tsx:7]: Avoid type assertions:

`it.each<[number, EventType, string]>([`

File: src/domain/eventSearch/searchResultList/__tests__/SearchResultsContainer.test.tsx [hunk 2]
```diff
@@ -1,23 +1,60 @@
...
+  [1, 'course', 'Valitsemillasi hakuehdoilla löytyi vain vähän hakutuloksia.'],
+  [4, 'course', 'Valitsemillasi hakuehdoilla löytyi vain vähän hakutuloksia.'],
+])(
+  'should return the proper results info text if %i %s results are found',
+  (eventsCount, eventType, infoText) => {
+    render(
+      <SearchResultsContainer
+        eventList={<div />}
+        eventsCount={eventsCount as number}
+        eventType={eventType as EventType}
```
Comment (Sartonon) [inline src/domain/eventSearch/searchResultList/__tests__/SearchResultsContainer.test.tsx:21]: These type assertions can be deleted

File: src/domain/eventSearch/searchResultList/__tests__/ResultsInfo.test.tsx [hunk 3]
```diff
@@ -1,18 +1,38 @@
...
+  expect(container).toMatchSnapshot();
+});
+
+test('events with 5 results matches snapshot of normal results', () => {
+  const { container } = render(
+    <ResultsInfo resultsCount={5} eventType="event" />
+  );
+
+  expect(container).toMatchSnapshot();
+});
```
Comment (Sartonon) [inline src/domain/eventSearch/searchResultList/__tests__/ResultsInfo.test.tsx:29]: I think 1 snapshot test might be enough here

**Review by Sartonon — CHANGES_REQUESTED** [review]
Looks good but couple snapshot tests could be deleted and also use generic type to avoid type assertions.

**Pull request merged**